### PR TITLE
LXD: Validate the host and port of endpoint

### PR DESCRIPTION
**Pull request by SimonRichardson**
## Description of change

The following commit ensures that we have a port for the lxd endpoint.
That way we help people coming to use the lxd-remote flow, will get
a better experience.

## QA steps

Bootstrap a remote host, this should add the default port 8443 if it's not supplied.

## Documentation changes

Users using the CLI should be able to add a URL without the port and it should use the default port to make the user experience of using lxd remoting easier.

## Bug reference

N/A

### CODE REVIEW

**Comment by SimonRichardson** [comment]
https://trello.com/c/g3MFaUmU/345-lxd-ux-ui-ensure-that-there-is-a-default-port-on-the-host

**Comment by SimonRichardson** [comment]
!!build!!

**Comment by pmatulis** [comment]
@SimonRichardson For documentation purposes, I'm going to need more background and specifics. What is this really about in layman's terms?
